pages/theming/css-framework.html

```diff
@@ -111,9 +111,9 @@
 <h4>Overlay &amp; Shadow</h4>
 <ul>
 	<li>
-		<code>.ui-widget-overlay</code>: Applies 100% width &amp; height dimensions to an overlay screen, along with background color/texture, and screen opacity.
```

Comment:
All themes that are used with UI 1.12 use the new default theme.css as template. So the changes in overlay and shadow apply to all themes.